src/PythonBindings/setup.py

```diff
@@ -10,7 +10,7 @@
     version='${SpECTRE_VERSION}',
     description="Python bindings for SpECTRE",
     author="SXS collaboration",
-    url="https://spectre-code.org",
+    url="${SPECTRE_HOMEPAGE}",
```

Comment:
I don't understand how the version and url work. Where does this ${....} come from? Or is this inserting something into CMake? Some documentation in the sense of code comments would be helpful.

Reply:
Either `@var@` or `${var}` work with cmake https://cmake.org/cmake/help/latest/command/configure_file.html. Changed to `@`-notation since we use that more.